runtime/src/transaction/mod.rs

```diff
@@ -0,0 +1,2 @@
+mod runtime_transaction;
```

Comment:
name's a little redundant now... :thinking:

Reply:
agree, i'll pond name after we settle on this comment https://github.com/solana-labs/solana/pull/34153#issuecomment-1817263881